src/Native/Unix/System.Security.Cryptography.Native/opensslshim.h

```diff
@@ -1039,6 +1039,8 @@ FOR_ALL_OPENSSL_FUNCTIONS
 
 #if OPENSSL_VERSION_NUMBER < OPENSSL_VERSION_1_0_2_RTM
 
+#define X509_CHECK_FLAG_NO_PARTIAL_WILDCARDS 4
```

Comment:
Would it be better to do an ifndef check for this and the others? Or that's either not possible or not better than the version number check?

Reply:
The others are functions, which can't be checked with if(n)def.  Since there's already a version check, the redundant ifndef doesn't seem worth it.